packages/dropdowns/stories/examples/Autocomplete/Default.tsx

```diff
@@ -66,7 +66,7 @@ export const Default: Story<IStoryProps> = ({
   showStartIcon
 }) => {
   const [selectedItem, setSelectedItem] = useState(options[0]);
-  const [inputValue, setInputValue] = useState('');
+  const [inputValue, setInputValue] = useState(options[0]);
```

Comment:
Update our example to ensure the initial selection (and subsequent selections, see line 98) that a sighted user would see always match the `<input>`s `value` that a screen reader would announce.